test/functionalTest/cases/1108_non_string_fields/non_string_fields.test

```diff
@@ -0,0 +1,194 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand that the restriction is in entity id and type, so probably we can do this renaming:

1108_non_string_fields/non_string_fields.test

to

1108_entity_id_and_type_non_string_fields/entity_id_and_type_non_string_fields.test

or something like that...